### DISCUSSION TITLE
access log 里面能增加流量数据么，每个请求的上传下载大小，类似 nginx 的 access log

### DISCUSSION THREAD

**Discussion by zhi1ong** [v2fly/v2ray-core#640] [Ideas]
Nginx 的 access.log 里面是比较详细的，包括每个请求的 body size，比较方便统计在不同网站的流量消耗；

v2ray 自带的流量统计只能看到总体的上传下载，access log 里也只有目标地址和 Email，不好做流量的分析，比方说有时候用的流量比较多的情况下，不好判断分网站、应用的消耗

**Comment by database64128** [comment]
`PR welcome`

**Reply by zhi1ong** [reply]
#642

**Comment by zhi1ong** [comment]
可以用 Goaccess 简单分析一下，再补点其他字段就更好了 😀

![image](https://user-images.githubusercontent.com/6677909/106230924-80dcc580-622b-11eb-9cca-02d92d485672.png)
